README.md

```diff
@@ -519,6 +519,9 @@ You first need to create an [Atlas account](https://atlas.hashicorp.com/account/
 * **github-token**: GitHub oauth token with `repo` permission.
 * **repo**: Repo slug, defaults to current one.
 * **target-branch**: Branch to push force to, defaults to gh-pages.
+* **keep-history**: Optional, create incremental commit instead of doing push force, defaults to false.
+* **allow-empty-commit**: Optional, defaults to false. Enabled if only keep-history is true.
```

Comment:
If there was some change, which won't affect anything in generated web-site it would trigger the build anyway. One may want to track that deployment actually took place and this would also indicate that web-site is up-to-date.
If nothing's changed after running deploy, then the deploy is either broken or changes didn't affect web-site. It might be as well a log of deployments in git and it would explicitly indicate that there were some changes in the main branch, while they didn't affect gh-pages.